handbook/ce/team-culture.md

```diff
@@ -0,0 +1,181 @@
+
+In the current model, the CE team can't easily "surge" to provide a lot of help to a single customer or in a single area, because every CE is already near capacity (20-30 assigned accounts per CE). As we encounter more complex customers and expand our use cases, the peak workload of even a single account will often exceed what a CE can provide even today.
+
+In the current model and in our current state, a customer paying us $1,800 a year may receive the same (or worse, more) time and attention than a customer paying us more than $200,000 a year. We believe that the ~80% of customers that are not "strategic" and their needs are better served by our team working on projects that indirectly help them (along with other teams, such as Customer Support) versus having only an assigned (but overloaded) CE to directly help them.
+
+#### Opportunities & Benefits
+Acknowledging that the above needs are important for our customers and our business, we believe that against all of the different teams that could work towards addressing these needs, CEs are uniquely positioned to be the best ones to do so. As such, we believe there exists an opportunity for CE -- a team that sits adjacent to so many different functions (Product, Engineering, Support, Marketing, Sales, Operations) internally and interfaces most deeply with our customers -- to expand our footprint and positively impact our customers and our business. A primary intention for accomplishing this is by enabling cross-account, project-based work.
+
+We believe that this is the exact moment in our journey to take such a risk and we bet that if we carve out focus and space, we will achieve the following benefits: matured capabilities (eg, demos, trials, data-driven customer health measurement and analysis, documentation, etc), increased footprint (eg, strategic account growth, dev community advocacy and outreach, building customer-centric solutions to expand Sourcegraph, etc), and internal opportunities for CE growth and advancement (eg, strategic focus, PM-type work, Eng-type build work, etc). 
+
+### Future Vision
+We envision our CE team being able to continue building and fostering relationships with developer users and champions at our prospects and customers, continue serving as the technical experts during the sales cycle and helping enable our customers throughout our journey together (from the very first meeting and every subsequent meeting thereafter), while being able to impact and deliver cross-functional, cross-customer projects and establish a new strategic customer focus.
+
+##### What we mean by cross-functional, cross-customer projects
+A project-based focused falls into two categories:
```

Comment:
Is this supposed to be "Project-based work" possibly? Or "The focus areas for a project-based CE"? "A project-based" feels like it was intended to have a noun after it.

Reply:
I don't see this. What line? If 118, how about just: "Project-based focused falls into two categories:"

Reply:
Sorry, yes, line 118: "A project-based focused falls into two categories" just seemed like it was missing the noun.

Reply:
```suggestion
Project-based work falls into two categories:
```

Reply:
Ah thank you for this change - you rock!!!